vtr_flow/scripts/download_ispd.py

```diff
@@ -68,39 +68,41 @@ def main():
         tar_gz_filename = "ispd_benchmarks_vtr_v" + args.version + '.tar.gz'
         md5_filename = "ispd_benchmarks_vtr_v" + args.version + '.md5'
         
-        tar_gz_url = urlparse.urljoin(ISPD_URL_MIRRORS[args.mirror], tar_gz_filename)
-        md5_url = urlparse.urljoin(ISPD_URL_MIRRORS[args.mirror], md5_filename)
+        tar_gz_url = urllib.parse.urljoin(ISPD_URL_MIRRORS[args.mirror], tar_gz_filename)
+        md5_url = urllib.parse.urljoin(ISPD_URL_MIRRORS[args.mirror], md5_filename)
 
-        external_md5 = load_md5_from_url(md5_url)
+        # Had to tack on a .decode() here as part of Python 2 to
+        # Python 3 conversion to convert from bytes to a string
```

Comment:
I'd probably leave the fact that this is a Python 2 to 3 conversion issue to the commit messages, and just leave the second line (converting from bytes to string) in the comment.